cli/src/commands/util.rs

```diff
@@ -230,6 +271,24 @@ fn cmd_util_config_schema(
     Ok(())
 }
 
+fn cmd_util_exec(args: &UtilExecArgs) -> Result<(), CommandError> {
+    let status = std::process::Command::new(&args.command)
+        .args(&args.args)
+        .spawn()
+        .map_err(|e| CommandError::new(CommandErrorKind::User, e))?
+        .wait()
+        .unwrap();
```

Comment:
nit: `.status()` can be used
https://doc.rust-lang.org/stable/std/process/struct.Command.html#method.status